nodetypes/http%3A%2F%2Fopentosca.org%2Fnodetypes/ApacheApp_w1/NodeType.tosca

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<Definitions xmlns="http://docs.oasis-open.org/tosca/ns/2011/12" xmlns:ns0="http://www.opentosca.org/winery/extensions/tosca/2013/02/12" xmlns:selfservice="http://www.eclipse.org/winery/model/selfservice" xmlns:testwineryopentoscaorg="http://test.winery.opentosca.org" targetNamespace="http://opentosca.org/nodetypes" id="nodeTypes-ApacheApp_w1">
+    <NodeType name="ApacheApp_w1" abstract="no" final="no" targetNamespace="http://opentosca.org/nodetypes">
+        <ns0:PropertiesDefinition elementname="properties" namespace="http://opentosca.org/nodetypes/propertiesdefinition/winery">
```

Comment:
Please also add the parent from which this NT is derived from. I guess `ToscaNormativeNodeTypes:WebApplication`?

Reply:
added